Frends.SFTP.WriteFile/Frends.SFTP.WriteFile.Tests/WriteTests.cs

```diff
@@ -0,0 +1,132 @@
+using NUnit.Framework;
+using System.Threading;
+using Frends.SFTP.WriteFile.Definitions;
+using Frends.SFTP.WriteFile.Enums;
+
+namespace Frends.SFTP.WriteFile.Tests;
+
+[TestFixture]
+class WriteTests : WriteFileTestBase
+{
+    [Test]
+    public void WriteFile_TestSimpleWrite()
+    {
+        var connection = Helpers.GetSftpConnection();
+        var input = new Input
+        {
+            Path = "/write/test.txt",
+            Content = "test",
+            FileEncoding = FileEncoding.ANSI,
+            WriteBehaviour = WriteOperation.Error
+        };
+
+        var result = SFTP.WriteFile(input, connection, new CancellationToken());
+        Assert.AreEqual("/write/test.txt", result.Path);
+        Assert.IsTrue(Helpers.DestinationFileExists(input.Path));
+        Assert.AreEqual("test", Helpers.GetDestinationFileContent(input.Path));
+    }
+
+    [Test]
+    public void WriteFile_TestWriteWithAppend()
+    {
+        var connection = Helpers.GetSftpConnection();
+        var input = new Input
+        {
+            Path = "/write/test.txt",
+            Content = "test",
+            FileEncoding = FileEncoding.ANSI,
+            WriteBehaviour = WriteOperation.Error
+        };
+
+        var result = SFTP.WriteFile(input, connection, new CancellationToken());
+        Assert.IsTrue(Helpers.DestinationFileExists(input.Path));
+
+        input.Content = "test";
+        input.WriteBehaviour = WriteOperation.Append;
+
+        result = SFTP.WriteFile(input, connection, new CancellationToken());
```

Comment:
You are not using the result-variable, so you can just make the method call without saving the returned value to a variable.